.ci/magician/cmd/vcr_merge.go

```diff
@@ -0,0 +1,142 @@
+package cmd
+
+import (
+	"fmt"
+	"magician/exec"
+	"magician/github"
+	"magician/source"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var vcrMergeCmd = &cobra.Command{
+	Use:   "vcr-merge",
+	Short: "Merge VCR cassettes",
+	Long: `This command is triggered in .ci/gcb-push-downstream.yml to merge vcr cassettes.
+
+	The command expects the following as arguments:
+	1. Reference commit SHA
+
+	It then performs the following operations:
+	1. Get the latest closed PR matching the reference commit SHA.
+	2. Run gsutil to list, copy, and remove the vcr cassettes fixtures.
+	`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I'd like us to move towards using RunE instead of Run + os.Exit (https://github.com/hashicorp/terraform-provider-google/issues/17479 for more details.)

Reply:
Done